plugins/slack_app/README.md

```diff
@@ -0,0 +1,11 @@
+# Slack App Plugin
+
+Here's what you need to do in order to turn on the Slack App plugin for your Samson.
+
+1. Go over to Slack and create a [new app](https://api.slack.com/apps/new). Many of the fields are completely up to you, but these are important:
+  - Grab the Client ID and secret and verification token, and record them in your environment as `SLACK_CLIENT_ID`, `SLACK_CLIENT_SECRET`, and `SLACK_VERIFICATION_TOKEN`.
```

Comment:
yes!

On Fri, Jul 15, 2016 at 2:13 AM, Zehua Liu <notifications@github.com> wrote:

> In plugins/slack_app/README.md
> <https://github.com/zendesk/samson/pull/1130#discussion_r70943366>:
>
> > @@ -0,0 +1,11 @@
> > +# Slack App Plugin
> > +
> > +Here's what you need to do in order to turn on the Slack App plugin for your Samson.
> > +
> > +1. Go over to Slack and create a [new app](https://api.slack.com/apps/new). Many of the fields are completely up to you, but these are important:
> > +  - Grab the Client ID and secret and verification token, and record them in your environment as `SLACK_CLIENT_ID`, `SLACK_CLIENT_SECRET`, and `SLACK_VERIFICATION_TOKEN`.
>
> Should we include these env var in .env.example?
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1130/files/f9ac922e76d375774c3b8681d0483f89608f4ed1#r70943366>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZyGGaGlVo5R6-h0JIuMPqD3ktJuPks5qV08kgaJpZM4JIRHk>
> .
>